p2p/raintree/network.go

```diff
@@ -246,21 +250,19 @@ func (n *rainTreeNetwork) GetBus() modules.Bus {
 	return n.bus
 }
 
+// Generate cryptographically secure random nonce
 func getNonce() uint64 {
```

Comment:
I left another comment about this elsewhere, but let's move out the helper into a shared utility under the crypto package.